src/mjolnir/pbfgraphparser.cc

```diff
@@ -65,6 +65,8 @@ struct graph_callback : public OSMPBF::Callback {
     infer_internal_intersections_ =
         pt.get<bool>("data_processing.infer_internal_intersections", true);
     infer_turn_channels_ = pt.get<bool>("data_processing.infer_turn_channels", true);
+    use_direction_on_ways_ = pt.get<bool>("data_processing.use_direction_on_ways", false);
```

Comment:
where does `reclassify_links` get set?

Reply:
https://github.com/valhalla/valhalla/blob/2fe491eea546eef74cd82fd1140a4689d33a61bb/src/mjolnir/graphbuilder.cc#L1168

After i enabled this I noticed that I want to move and parser, builder, or enhancer flags to under the `data_processing` section in the config.